orion/sites/admin/src/programs/program.js

```diff
@@ -49,8 +77,16 @@ class ProgramRow extends React.Component {
   render() {
     const row = this.props.row;
     const url = "/program/" + row.programId + "/edit";
+    let isUnpublished = true;
+    let checkbox = <div></div>;
+    if (isUnpublished){
+        checkbox = (<input className="li-small" type="checkbox" name="unpublished" onClick={this.onClickBox}/>);
+    } else {
+        checkbox = (<div className="li-small"></div>);
+    }
```

Comment:
very nice!

Reply:
Consider moving this part into a function.
So in render(), you call this:
```
let checkbox = renderCheckbox(true);
```

Somewhere else outside the React Component, the function looks like:
```
function renderCheckbox(isUnpublished) {
    let checkbox = ...
    if (isUnpublished) {...} else {...}
    return checkbox;
}
```

Then in the very near future, instead of just passing true or false, it can be a variable!